client/app/views/templates/view_steps_agreement.jade

```diff
@@ -2,4 +2,7 @@ extends view_base
 
 block region
     h1=t('Conditions générales')
```

Comment:
We must use key strings for localization. Do we doit here, or do your prefer to open another PR to declare those string keys?